conda-forge.yml

```diff
@@ -1,11 +1,10 @@
 bot:
   automerge: true
 build_platform:
-  linux_aarch64: linux_64
   osx_arm64: osx_64
 conda_forge_output_validation: true
 provider:
-  linux_aarch64: default
+  linux_aarch64: azure
```

Comment:
I misinterpreted your comment and thought the default, no emulation, was preferred to fix this. Thanks!